### PR TITLE
Add new website

### PR DESCRIPTION
**Pull request by datapythonista**
Proposing a new website, based on how the pandas one is built, to help organize the information, and make it easier for new users to understand what Ibis is about.

It can be seen here: https://datapythonista.me/ibis/

This PR only has the content (not the script to build the website), so we focus on the content. I'll follow up with a PR to build it an publish it, if we move forward with this.

### CODE REVIEW

**Comment by datapythonista** [comment]
Thanks @xmnlab those are good points.

>     1. in the desktop, the gray box in the first page seems too big (maybe because the text is very short).
>        would be better if this box would be smaller?

Yes, it probably doesn't look great. Let's see the feedback about the content, and if people is happy with this, I guess an option is to replace the grey in the background by a cool image. Or add a sidebar, like in https://pandas.pydata.org/



>     2\. would this new documentation be available at `https://docs.ibis-project.org/` ? if yes, maybe the documenation link need to point to an endpoint e.g. `https://docs.ibis-project.org/docs`

That's something that needs to be decided. My preference would be to use simply https://ibis-project.org and https://ibis-project.org/docs

**Comment by jreback** [comment]
can you add a link from the docs back to the web; is the idea to migrate docs to web as well?

**Comment by datapythonista** [comment]
For the docs, I want to try to render them with Sphinx, but using the website layout. So for the user everything is the same, even if they are generated with different programs.

But all the changes are too big for a single PR. So far just adding the content for the web. Follow up PRs I have in mind:
- Integration web and docs
- Restructure of the docs
- Review and improve tutorials
- Links to use the tutorials on binder
- Publish the new web from the CI

Any feedback on the whole plan welcome. Comments on the content of this PR also welcome (not an Ibis expert myself, so surely some things can be improved).

File: docs/web/about/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Enable data analysts to translate local, single-node data idioms to scalable
+  computation representations (e.g. SQL or Spark)
+- Integration with pandas and other Python data ecosystem components
+- Provide high level analytics APIs and workflow tools to enhance productivity
+  and streamline common or tedious tasks.
+- Integration with community standard data formats (e.g. Parquet and Avro)
+- Abstract away database-specific SQL differences
+
+As the [Apache Arrow](http://arrow.apache.org/) project develops, we will
```
Comment (saulshanabrook) [inline docs/web/about/index.md]: This paragraph confuses me a bit. It seems to be suggesting a future direction that somehow uses arrow to help execute in Spark and Impala?

Comment (datapythonista) [reply docs/web/about/index.md]: This page is a copy of the beginning of the current documentation index.

Happy to update it here if it makes sense. I don't really know if it's still relevant or not.

Comment (saulshanabrook) [reply docs/web/about/index.md]: Ah cool, I thought you just wrote this so was hoping you had more insight. No worries, can always be changed later.

**Comment by datapythonista** [comment]
@jreback if you don't have comments on the content here, can we get this merged? I can work on rendering the this new website and integrate it with the docs this week.

Btw, do you know who is the admin of ibis-project.org? Not in this PR, but I think it makes sense to use the bare domain once we merge the follow up that publishes the website.

**Comment by jreback** [comment]
ci is broken so until that is fixed not merging anything

you are welcome to have a look

**Comment by datapythonista** [comment]
Same here, also green @jreback

**Comment by jreback** [comment]
thanks @datapythonista

**Comment by jreback** [comment]
certainly followups to fix context / layout are more than welcome :->

**Comment by datapythonista** [comment]
> certainly followups to fix context / layout are more than welcome :->

Sure, the web is not being rendered, I'll follow up with the rendering, and several other fixes. But it's being helpful to have this merged, so the other PRs are not massive. Thanks!

**Comment by wesm** [comment]
Thanks for working on this, having a better website for Ibis is long overdue

**Pull request merged**